examples/siro_sandbox/sandbox_app.py

```diff
@@ -1089,6 +1100,14 @@ def parse_debug_third_person(args, framebuffer_size):
 
     driver = SandboxDriver(args, config, gui_app_wrapper.get_sim_input())
 
+    # sanity check if there are no agents with camera sensors
+    # if (
+    #     len(config.habitat.simulator.agents) == 1
+    #     and args.gui_controlled_agent_index is not None
+    # ):
+    #     assert driver.get_sim().createRenderer == False
```

Comment:
AttributeError: 'RearrangeSim' object has no attribute 'createRenderer'.

Reply:
Thank you!

Here are the values:
<img width="405" alt="Screenshot 2023-05-26 at 12 09 36" src="https://github.com/facebookresearch/habitat-lab/assets/37709869/e4ebbbe7-6357-41b9-944b-8dc49c0081e5">